LayoutTests/imported/w3c/web-platform-tests/xhr/send-redirect.htm

```diff
@@ -9,20 +9,28 @@
   <body>
     <div id="log"></div>
     <script>
-      function redirect(code) {
-        var test = async_test(document.title + " (" + code + ")")
+      function redirect(code, method="GET", redirectLocation="content.py") {
```

Comment:
That's great, please make sure to upstream to WPT or it will get lost during the next re-sync.

Reply:
https://github.com/web-platform-tests/wpt/pull/41417